about.html

```diff
@@ -0,0 +1,157 @@
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                    <li>
+                        <div class="close-icon">
+                            <i class="fa fa-times fa-3x" aria-hidden="true"></i>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+
+            <nav class="menu-bigscreen">
+                <a href="index.html" class="logo">
+                    <img src="assets/apple-touch-icon.png" alt="logo">
+                </a>
+                <ul>
+                    <li>
+                        <a href="index.html" class="a-links">Home</a>
+                    </li>
+                    <li>
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                </ul>
+
+            </nav>
+        </header>
+
+        <div id="abt-container">
+            <P class="p-3">Unleashing the Power of Sports in India</P>
+            <h1 class="head-1">Sport Summit <br> India 2023</h1>
+            <p class="box-2">
+                India has always had a rich history and culture in sports, from ancient games like Kabaddi and Kho Kho
+                to modern-day cricket and football. However, the potential of sports to transform the country goes
+                beyond just entertainment and recreation. The Sports Summit India aims to explore and harness the power
+                of sports to empower the nation in various ways. </p>
+           <P class="email-me">Please contact us per Email for any further questions about Sport Summit India 2023!</P>
+           <p class="email-id">ss2023@india.org</p>
+        </div>
+ 
+    </section>
+
+    <section id="venue">
+        <div class="ven">
+            <h2 class="ven-text">Venue</h2>
+            <hr class="short-hr">
+            <h3 class="venue"><i class="fa fa-map-marker" aria-hidden="true"></i> Indira Gandhi Stadium Complex, New
+                Delhi</h3>
+                <div class="image-overlay-0">
+                    <img src="assets/Indira-Gandhi-s29090.jpg" alt="img" class="stand">
+                    <span class="overlay-text-0">2023</span>
+                </div>
```

Comment:
- [ ] Kindly ensure that this part of the page has the proper content as the design, It should contain the logo and some declaration.

- Your design
<img width="357" alt="image" src="https://user-images.githubusercontent.com/41163571/235315391-d15d9233-8f69-47ee-a2b2-aea8b22990ca.png">

- Behance design
<img width="471" alt="image" src="https://user-images.githubusercontent.com/41163571/235315424-e3a4f030-78bc-4de6-bc60-0915e1cf92ad.png">